common/scripted_effects/00_scripted_effects.txt

```diff
@@ -19044,11 +19015,9 @@ china_war_spawn_troops_no_disband_effect = {
 					home = event_target:spawn_province
 					troops = {
 						archers = { 122 122 }
-						light_infantry = { 71 71 }
-						heavy_infantry = { 85 85 }
-						pikemen = { 78 78 }
```

Comment:
I would summarize heavy and light infantry... or even give mages or something else.